doc/service-tests.md

```diff
@@ -162,11 +162,19 @@ harness will call it for you.
 Run the test:
 
 ```
-npm run test:services -- --only=travis
+npm run test:services
 ```
 
-The `--only=` option indicates which service or services you want to test. You
-can provide a comma-separated list of ids.
+The test runner examines the commit history to automatically detect which
+services to test.
+
+If for any reason it's not detecting the correct services, you can specify them
+yourself:
+
+npm run test:services -- --only=travis
+
```

Comment:
We should add a note to the docs here that we can also run `npm run test:services -- --all` to run the full service test suite.